spec/config_skeleton_spec.rb

```diff
@@ -68,16 +68,19 @@ def instance
     thread.join(2)
   end
 
-  it "watches for file changes and regenerates config on change" do
-    thread = Thread.new { runner.run }
-    wait_until { File.read(cfg_file_path) != "" }
-    config_before_file_change = File.read(cfg_file_path)
-    File.write(tmp_watch_file, "some data")
-    config_after_file_change = nil
-    wait_until { config_after_file_change = File.read(cfg_file_path) != config_before_file_change }
-    expect(config_before_file_change).not_to eq(config_after_file_change)
-    ultravisor.shutdown
-    thread.join(2)
+  # No inotify, no watching for file changes
+  if !ENV["DISABLE_INOTIFY"]
```

Comment:
Could we use `skip()` here instead, so that a message is printed to the console and the user is aware it has been skipped? (also means we don't have to change indentation)

Reply:
Done!